<launch>
  <node pkg="moveit_check" type="object_position_server.py" name="object_position_server" output="screen" />
  <node pkg="moveit_check" type="object_position_right_hand_server.py" name="object_position_right_hand_server" output="screen" />
</launch>
